tests/aws/lambda/determineEventSource.test.js

```diff
@@ -0,0 +1,37 @@
+import test from 'tape';
+
+import determineEventSource, { AWS_EVENT } from '../../../src/aws/lambda/determineEventSource';
+
+import mockEventCloudfront from './sample-events/cloudfront.json';
+import mockEventApiGateway from './sample-events/apigateway-proxy.json';
+import mockEventCognitoSync from './sample-events/cognito-sync.json';
+import mockEventDynamoDB from './sample-events/dynamodb.json';
+import mockEventIotButton from './sample-events/iot-button.json';
+import mockEventLex from './sample-events/lex.json';
+import mockEventKinesis from './sample-events/kinesis.json';
+import mockEventKinesisFirehose from './sample-events/kinesis-firehose.json';
+import mockEventS3 from './sample-events/s3.json';
+import mockEventScheduled from './sample-events/scheduled-event.json';
+import mockEventSes from './sample-events/ses.json';
+import mockEventSns from './sample-events/sns.json';
+
+function eventTypeMatches(event, expectedEvent) {
```

Comment:
@acastro simplify to `const eventTypeMatches = (event, expected) => determineEventSource(event) === expected;